hardware_interface/include/hardware_interface/types/hardware_interface_return_values.hpp

```diff
@@ -15,11 +15,25 @@
 #ifndef HARDWARE_INTERFACE__TYPES__HARDWARE_INTERFACE_RETURN_VALUES_HPP_
 #define HARDWARE_INTERFACE__TYPES__HARDWARE_INTERFACE_RETURN_VALUES_HPP_
 
+#include <cstdint>
+
 namespace hardware_interface
 {
-using hardware_interface_ret_t = int;
-static constexpr hardware_interface_ret_t HW_RET_OK = 0;
-static constexpr hardware_interface_ret_t HW_RET_ERROR = 1;
+enum class return_type : std::uint8_t
+{
+  OK = 0,
+  ERROR = 1,
+
+  CLAIMED_ERROR = 10,
+  ALREADY_CLAIMED = 11,
+  NOT_CLAIMED = 11,
+  UNATHORIZED_UNCLAIM = 13,
+  NON_CLAIMED_WRITE = 15,
+
+  CAN_NOT_READ = 20,
```

Comment:
Where are these values being used?

Reply:
Currently nowhere... but I am planing to use them in actuators. Should I remove this Andrade when it is used?